macos/CountlyTestApp-macOS/AppDelegate.m

```diff
@@ -16,7 +16,7 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification
     config.host = @"https://YOUR_COUNTLY_SERVER";
     config.enableDebug = YES;
 
-//  config.features = @[CLYPushNotifications];              //Optional  push notification features
+//  config.features = @[CLYFeature.pushNotifications];              //Optional  push notification features
```

Comment:
For Objective c code don't need to change it, it will not work for Objective C
Its only needed for Swift

Reply:
done